lib/auth/legacy.js

```diff
@@ -25,7 +25,7 @@ module.exports.login = function login (creds, registry, scope, cb) {
       return profile.adduser(username, email, password, {registry: registry})
     }).catch((err) => {
       if (err.code === 'EOTP' && !auth.otp) {
-        return read.otp('Authenicator provided OTP:').then((otp) => {
+        return read.otp('Authenticator provided OTP:').then((otp) => {
```

Comment:
Ouch, I thought I'd gotten all of those typos. My fingers really don't like the `t` in that word.